var/spack/repos/builtin/packages/py-fastapi/package.py

```diff
@@ -13,13 +13,19 @@ class PyFastapi(PythonPackage):
     homepage = "https://github.com/tiangolo/fastapi"
     pypi = "fastapi/fastapi-0.88.0.tar.gz"
 
+    version("0.98.0", sha256="0d3c18886f652038262b5898fec6b09f4ca92ee23e9d9b1d1d24e429f84bf27b")
     version("0.88.0", sha256="915bf304180a0e7c5605ec81097b7d4cd8826ff87a02bb198e336fb9f3b5ff02")
 
     variant("all", default=False, description="Build all optional dependencies")
 
+    depends_on("py-hatchling@1.13:", when="@0.98:", type="build")
     depends_on("py-hatchling", type="build")
```

Comment:
Is there a benefit to ordering the specific `py-hatchling@1.13` before `py-batchling`? I've often seen the reverse in packages and just curious how we should be standardizing.

Reply:
My standardization is to sort from newest to oldest requirements, just like we sort versions from newest to oldest. It helps when vertically scanning for dependencies from top to bottom as well.